package/koreader/package

```diff
@@ -36,18 +40,7 @@ package() {
     ln -s /opt/bin/fbdepth "$pkgdir"/opt/koreader/fbdepth
 
     install -D -m 644 -t "$pkgdir"/opt/etc/draft/ "$recipedir"/koreader.draft
+    install -D -m 644 -t "$pkgdir"/opt/usr/share/applications/ "$recipedir"/KOReader.oxide
     install -D -m 644 -t "$pkgdir"/opt/etc/draft/icons/ "$srcdir"/resources/koreader.png
     install -D -m 755 -t "$pkgdir"/opt/bin/ "$recipedir"/koreader
```

Comment:
You should use `$srcdir` here, since you added the files to the `source` array.
```suggestion
    install -D -m 644 -t "$pkgdir"/opt/etc/draft/ "$srcdir"/koreader.draft
    install -D -m 644 -t "$pkgdir"/opt/usr/share/applications/ "$srcdir"/KOReader.oxide
    install -D -m 644 -t "$pkgdir"/opt/etc/draft/icons/ "$srcdir"/resources/koreader.png
    install -D -m 755 -t "$pkgdir"/opt/bin/ "$recipedir"/koreader
```